build.savant

```diff
@@ -19,7 +19,7 @@ jakartaXMLBindVersion = "4.0.0"
 slf4jVersion = "2.0.7"
 testngVersion = "7.8.0"
 
-project(group: "io.fusionauth", name: "fusionauth-samlv2", version: "0.11.0", licenses: ["ApacheV2_0"]) {
+project(group: "io.fusionauth", name: "fusionauth-samlv2", version: "0.11.1", licenses: ["ApacheV2_0"]) {
```

Comment:
This is technically a behavior change, I would bump the minor instead of the patch

Reply:
It's an internal behavior change that does not impact the public APIs. I think that qualifies as a patch version, doesn't it?

Reply:
I think maybe wires got crossed. This version compared to the previous `0.11.0` version only resolves a bug where the SAML assertion decryption was not falling back to default algorithm parameters if they were not specified in XML. `0.11.0` was the version that added initial support for parsing encrypted assertions, and it was released as a new minor revision.